.github/workflows/build-common.yml

```diff
@@ -267,6 +267,14 @@ jobs:
           # gradle enterprise is used for the build cache
           gradle-home-cache-excludes: caches/build-cache-1
 
+      - id: read-build-scan
+        name: Read build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo "build-scan=$(cat build-scan.txt)" >> $GITHUB_OUTPUT
+      - name: Build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo ${{ steps.read-build-scan.outputs.build-scan }}
```

Comment:
why 2 steps instead of 1?

Reply:
The sample code I found from google used 2 steps. I was intending on relying on you pointing out a simpler solution if one exists.

Reply:
maybe the $GITHUB_OUTPUT is displayed in some summary box?